galata/src/galata.ts

```diff
@@ -207,7 +207,7 @@ export namespace galata {
     tmpPath: string,
     waitForApplication: (page: Page, helpers: IJupyterLabPage) => Promise<void>
   ): Promise<IJupyterLabPageFixture> {
-    const context = await browser.newContext();
+    const context = await browser.newContext({ acceptDownloads: true });
```

Comment:
Actually it looks like we might not be using this `newPage` helper at the moment.

Reply:
Indeed it is not.

i think the better way is to use the test options: https://playwright.dev/docs/api/class-testoptions#test-options-accept-downloads
And therefore updating https://github.com/jupyterlab/jupyterlab/blob/master/galata/src/playwright-config.ts

Reply:
Ah nice, thanks for the hint!